cfgov/data_research/handlers.py

```diff
@@ -0,0 +1,82 @@
+import json
+from govdelivery.api import GovDelivery
+from urlparse import urlparse
+
+from django.conf import settings
+from django.contrib import messages
```

Comment:
Will this actually work properly on the production site? I know it's in use elsewhere (like in [`cal`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cal/views.py#L7)) but by default this functionality uses [sessions](https://docs.djangoproject.com/en/1.8/topics/http/sessions/), which requires cookies. I thought that cookies can't work properly with Akamai in production. @sebworks @rosskarchner